2014-15/C08 Match on CHI variables - Cohorts SPARRA and HHG.sps

```diff
@@ -11,10 +11,10 @@
 
 **********************************************************************************************************.
 
-match files file = !File + "temp-source-episode-file-6-" + !FY + ".zsav"
-    /table = !File + "Demographic_Cohorts_" + !FY + ".zsav"
-    /table = !File + "Service_Use_Cohorts_" + !FY+ ".zsav"
-    /table = !Extracts_Alt + "SPARRA/SPARRA-20" + !NextFY + ".zsav"
+match files file = !Year_dir + "temp-source-episode-file-6-" + !FY + ".zsav"
+    /table = !Cohort_dir + "Demographic_Cohorts_" + !FY + ".zsav"
+    /table = !Cohort_dir + "Service_Use_Cohorts_" + !FY+ ".zsav"
```

Comment:
Check the space on this line (looks like it didn't have one before either so maybe it's not actually nessecary)